app/src/main/java/com/kickstarter/ui/activities/LoginActivity.kt

```diff
@@ -5,6 +5,7 @@ import android.content.Intent
 import android.os.Bundle
```

Comment:
Working perfectly
The only scenario I haven't test has been log in with facebook, the other user paths works perfectly, even the 2fa